l10n_fr_hr_rup/models/hr_contract.py

```diff
@@ -14,7 +14,6 @@ class HrContract(models.Model):
 
     @api.onchange("employee_id")
     def _onchange_employee_id(self):
-        super()._onchange_employee_id()
```

Comment:
**Alternative 1: onchange**

```suggestion
        res = None
        if hasattr(super(), "_onchange_employee_id"):
            res super()._onchange_employee_id()    # pragma: no cover
```
and `return res` at the end.

This is to play nice with other modules that may also be defining an `_onchange_employee_id` method.
The name is generic enough, so one has to be careful.

-----

**Alternative 2: computed stored writeable field** :star2: _Preferred soltution_ 
Convert `pcs_id`, `qualification`, `work_location` into computed fields with `store=True` and `readonly=False`. each with it's own `_compute_$field_name` method depending on `@api.depends("employee_id")`

Reply:
Thanks for the comment @ivantodorovich !
Ok to go for alternative 2.
Just one question : why is it better to use 3 different `_compute` methods instead of only one (the same for the 3 fields) ?

Reply:
Technically in this case there isn't any difference. But I find it's better for extendability, so in general it's a good practice.

My rule of thumb is to always use different compute methods, with the only exception being the case were there's a performance gain in grouping.

Reply:
done ! https://github.com/OCA/l10n-france/pull/294/files#diff-058c69791d50b8616ac786f6e1aa76ae3338117cae45400e9cab2510bd6dc0f8R11

Thanks again for the review !